### PR TITLE
2.1.0 release notes 1651742401

### PR DESCRIPTION
**Pull request by ablease**
Please wait for approval from my team first before merging.

### CODE REVIEW

**Review by coro — CHANGES_REQUESTED** [review]
Release notes look good, just a couple of version changes then LGTM

File: releases.html.md.erb [hunk 1]
```diff
@@ -25,20 +38,21 @@ This release has the following issues:
...
   </tr>
   <tr>
     <td>HAProxy</td>
-    <td>1.8.29</td>
+    <td>1.8.30</td>
   </tr>
   <tr>
     <td>OSS RabbitMQ<sup>*</sup></td>
-    <td>3.8.14</td>
+    <td>3.8.29</td>
```
[the remark this answers is not in the document]
Comment (ablease) [reply releases.html.md.erb]: Thank you for spotting this, resolved.

File: releases.html.md.erb [hunk 2]
```diff
@@ -7,13 +7,26 @@ owner: London Services
 
 ## <a id="2-0-x"></a> v2.0.x
```
[the remark this answers is not in the document]
Comment (ablease) [reply releases.html.md.erb]: Thanks for spotting this, resolved.

**Comment by ablease** [comment]
OK this can be merged now. 👍

**Pull request merged**